packages/cypress/cypress/integration/common/refapp-3.x/common.js

```diff
@@ -0,0 +1,13 @@
+import {Given, And} from '@badeball/cypress-cucumber-preprocessor';
+
+Given('the user login to the Outpatient Clinic', () => {
+    cy.on('uncaught:exception', (err, runnable) => {
+      return false;
+    });
+      cy.login();
+      cy.visit('home');
+  })
+  
+And('the user arrives on a patient’s chart page', () => {
+    cy.visit(`patient/${patient.uuid}/chart`);
+});
```

Comment:
@jayasanka-sack I think moving the before hooks from here to their corresponding feature files is what you emphasised on yesterday’s call.  

If there is somethings else, kindly let me know.